pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -29,6 +31,13 @@ buildPythonPackage rec {
     export BERKELEYDB_DIR=${pkgs.db.dev};
   '';
 
+    postPatch = ''
+    substituteInPlace test3.py \
```

Comment:
Indent once the line 35, look the rest of the file.

Reply:
Is this good now ?